builder/solver_monitor.go

```diff
@@ -519,8 +519,8 @@ func (sm *solverMonitor) reprintFailure(errVertex *vertexMonitor, phaseText stri
 	errVertex.printError()
 }
 
-var vertexRegexp = regexp.MustCompile("^\\[([^\\]]*)\\] (.*)$")
-var targetAndSaltRegexp = regexp.MustCompile("^([^\\(]*)(\\(([^\\)]*)\\))? (.*)$")
+var vertexRegexp = regexp.MustCompile("(?s)^\\[([^\\]]*)\\] (.*)$")
```

Comment:
what does `(?s)` match? I'm had a quick search for `(?` under https://github.com/google/re2/wiki/Syntax and couldn't figure it out.

Reply:
It's a regex flag to include `\n` in `.` matches. I had an issue with the multi-line script not showing up in the output because it was classified as internal, because one of these two regexes didn't match.

See `(?flags)` here https://golang.org/pkg/regexp/syntax/ under `Grouping`.

Reply:
`s              let . match \n (default false)`; ah ha! thanks for pointing that out. makes more sense now.